include/WCSimTrajectory.hh

```diff
@@ -112,6 +113,12 @@ public: // with description
     return bTypes;
   }
 
+// Functions to set/get photon history
+  inline void AddPhotonScatter(G4int val) { pScatter += val; }
+  inline void AddPhotonReflection(G4int val) { pReflec.push_back(val); }
```

Comment:
Why not use `ReflectionSurface_t` for these functions? Similarly for the member variables